### DISCUSSION TITLE
Typos in Prediction > Version 2 (Experimental) > Getting Started

### DISCUSSION THREAD

**Discussion by Rikougo** [FirstGearGames/FishNet#625] [Documentation]
- Found some typos in CSPv2 page in [Example Script section](https://fish-networking.gitbook.io/docs/manual/guides/prediction/version-2-experimental/getting-started#example-script) section the `ReconcileData` constructor takes `PredictedRigidbody` type that should be `PredictionRigidbody` (comment on PredictionRigidbody field also mention PredictedRigidbody). also found in [Networked Non-Controlled Physics page](https://fish-networking.gitbook.io/docs/manual/guides/prediction/version-2-experimental/getting-started/networked-non-controlled-physics)
- All attributes use V2 suffix that doesn't exists anymore.

**Comment by FirstGearGames** [comment]
Ty! I'll get that fixed right away. I was in a hurry to update much as I
could at the end of the day.

**Comment by FirstGearGames** [comment]
I believe I got them all. Thank you again!
